specs/LoaderDllFolderPath.md

```diff
@@ -0,0 +1,47 @@
+# Background
+The WebView2 loader code is what knows where to find and start or connect to already running WebView2 runtimes. This can either be statically linked for C based projects, or is also available as a standalone DLL. For .NET projects they must use the DLL since the loader code is native and we don't have a way to merge the native code into a managed module. Before this change the .NET WebView2 API module would look for the WebView2Loader.dll in the its same folder. But some .NET projects have requirements about where they can place DLLs and so this property is introduced to allow end developers to place the WebView2Loader.dll in any folder and specify the path of 'WebView2Loader.dll' explicitly.
+So we add an API to enable them to specify the path, so that they can use 'WebView2Loader.dll' from any path.
+
+# Description
+We add new static `LoaderDllFolderPath` property in `CoreWebView2Environment` class.
+The property allow the end developers to specify the folder's path containing `WebView2Loader.dll`.
+
+# Examples
+## C#
+```c#
+Task<CoreWebView2Environment> CreateEnvironmentAsync()
+{
+    // Specify a relative path '.'. There should be a 'WebView2Loader.dll' file in the folder.
+    CoreWebView2Environment.LoaderDllFolderPath = ".";
```

Comment:
Does "module would look for the WebView2Loader.dll in the its same folder" mean that "." is the default and initial value of this string?  If so, sample should show use of a non-default value.

Reply:
This is a  relative paths , and relative paths are relative to the path of the `Microsoft.Web.WebView2.Core.dll` module.
This is not the default path, such as if the user call SetDllDirectory, the default path will be the setted by SetDllDirectory.

Reply:
Does this mean the default value is empty string "", and you can clear/reset LoaderDllFolderPath by assigning empty string?

Reply:
Default value is empty string "", or do not set. And if we have not call the function in the loader.dll(the dll has not been loaded), we can clear/reset LoaderDllFolderPath by assigning empty string. Otherwise we cannot reset LoaderDllFolderPath.

Reply:
If the property isn't set then we get the existing behavior of looking in the same path as the Microsoft.Web.WebView2.Core.dll. So effectively the default is sort of '.'. At any rate we need to update the documentation to state what the default value and default behavior is.